src/core/constructs/output_manager.py

```diff
@@ -23,45 +23,23 @@
 
 
 class OutputManager:
-
     def __init__(self, console: Console = None, progress: Progress = None) -> None:
 
-        if console:
-            self._console = console
-        else:
-            self._console = Console()
-
-        if progress:
-            self._progress = progress
-        else:
-            self._progress = None
+        self._console = console or Console()
```

Comment:
I did not know that you can do this in Python. I like it a lot more than the above `if` statements!